```diff
diff --git a/etc/gamescope-session.conf b/etc/gamescope-session.conf
new file mode 100644
index 000000000000..93b9f63e1e51
--- /dev/null
+++ b/etc/gamescope-session.conf
@@ -0,0 +1,9 @@
+# gamescope-session
+
+#OUTPUT_CONNECTOR="whatever default value"
+#FOO="BAR"
+
+
+# session select
+
+#DESKTOP_SESSION="plasma.desktop"
diff --git a/usr/lib/steamos/gamescope-session b/usr/lib/steamos/gamescope-session
index dcab16afe887..e02cc1b436da 100755
--- a/usr/lib/steamos/gamescope-session
+++ b/usr/lib/steamos/gamescope-session
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+source /etc/gamescope-session.conf || echo "There was a problem sourcing the configuration file"
+
 # Create run directory file for startup and stats sockets
 #   shellcheck disable=SC2030 # (broken warning)
 tmpdir="$([[ -n ${XDG_RUNTIME_DIR+x} ]] && mktemp -p "$XDG_RUNTIME_DIR" -d -t gamescope.XXXXXXX)"
 ```
 
 So everything is commented initially with the default values, and defaults will be populated in each script (gamescope-session, steamos-session-select). Of course, I'm open to alternatives if the implementation is "better".

--- a/usr/lib/steamos/gamescope-session
+++ b/usr/lib/steamos/gamescope-session
@@ -253,5 +253,5 @@ exec gamescope \
 	--fade-out-duration 200 \
 	--cursor-scale-height 720 \
 	-e -R "$socket" -T "$stats" \
-	-O "$output_connector" \
+	-O "${OUTPUT_CONNECTOR:-*,eDP-1}" \
 

